packages/eslint-config-react/package.json

```diff
@@ -89,6 +89,5 @@
     "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0"
-  },
-  "prettier": "@yoo-digital/prettier"
+  }
```

Comment:
sadness :D why remove prettier?

Reply:
already defined in the root package.json, seems to be enough